src/hooks/useAuthorityLinking/useAuthorityLinking.js

```diff
@@ -87,12 +116,17 @@ const useAuthorityLinking = ({ tenantId, marcType, action } = {}) => {
       }
     });
 
+    if (linkingRule.subfieldModifications?.length) {
+      // change the order only for the subfields from `subfieldModifications` to the same order as in the authority record.

```

Comment:
We shouldn't have the same order as in authority, but the order of subfields in a linking rule. The only exception is that for bib 240 we need to add the t->a subfield first. The rest shouldn't change.
A general approach for handling ordering of subfield modifications will be investigated and discussed additionally

Reply:
Thanks, this made the task easier.